vnext/PropertySheets/OutputMSBuildProperties.targets

```diff
@@ -18,7 +18,8 @@
         "WindowsTargetPlatformVersion": "$(WindowsTargetPlatformVersion)",
         "UseExperimentalNuGet": "$(UseExperimentalNuGet)",
         "UseHermes": "$(UseHermes)",
-        "UseWinUI3": "$(UseWinUI3)"
+        "UseWinUI3": "$(UseWinUI3)",
+        "UseFabric": "$(UseFabric)",
```

Comment:
Even though `RnwNewArch` will always be empty in pre 0.76 builds, I'd rather it still be here and still be reported as such.